device_test.go

```diff
@@ -44,6 +44,28 @@ func TestClaimAndRelease(t *testing.T) {
 		t.Fatalf("OpenDeviceWithVIDPID(0x8888, 0x0002): %v", err)
 	}
 
+	mfg, err := dev.Manufacturer()
```

Comment:
also add a call to Manufacturer after dev was Close()d, expecting an error.

Reply:
done